### ISSUE TITLE
Breaking change to controlled Select components in 5.7.2

### ISSUE THREAD

**Issue by robcaldecottvelo** [jamesmfriedman/rmwc#552]
**RMWC 5.7.2**

This sandbox is using 5.7.0 of the Select component:

https://codesandbox.io/s/rmwc-select-570-vpqcb

Both Selects start with the label as a placeholder and a blank value.

Now select a Country and then select a Region. Then select a different country: the Region should reset.

There are some tests you can run to verify this behaviour.

Now try this version which using 5.7.2 of the Select:

https://codesandbox.io/s/rmwc-select-v572-0h8jp

Now the Select components do not initially render correctly and the tests fail: it looks like initialising them with an empty string as the value no longer works in 5.7.2. I have tried adding a `placeholder=""` prop but this makes no difference.

Do I have to include a blank option to make this work?

**Comment by anirban09** [comment]
Hey, we ran into this issue too, a month ago.

The change required to make this work is to not set the values initially, i.e. initialize `country` and `region` as `undefined` instead of `""`.

```diff
-  const [country, setCountry] = React.useState("");
-  const [region, setRegion] = React.useState("");
+  const [country, setCountry] = React.useState();
+  const [region, setRegion] = React.useState();
```

This is due to PR #535, which fixed #533.

**Comment by robcaldecottvelo** [comment]
Thanks @anirban09. This is going to play havoc with our existing code (where we have shared input/select code) and tests, plus using `undefined` as a value for a controlled component doesn't smell right. We'll have to stay on 5.7.0.

**Comment by jamesmfriedman** [comment]
Thanks for the bug report. Remember you can ask pun the Select to 5.7.0 and upgrade the rest of the library as a workaround for now. That’s probably the easier play instead of refactoring, although I am going to spend quite a bit of time on bug fixes coming up.

**Comment by jamesmfriedman** [comment]
For the record, these continue to be the absolute bane of my existence. mdc-web has completely refactored them 4 separate times, and continue to go back and forth on their decisions to support native selects...

So first of all, I'm definitely not ditching native selects because that would be awful. Second, here is a working sandbox with the next alpha. https://codesandbox.io/s/rmwc-select-v572-820ve

**Comment by jamesmfriedman** [comment]
Fixed in 6.0.0
